cfme/configure/access_control.py

```diff
@@ -77,7 +77,9 @@ class DetailsUserView(ConfigurationView):
     def is_displayed(self):
         return (
             self.title.text == 'EVM User "{}"'.format(self.context['object'].name) and
-            self.accordions.accesscontrol.is_opened
+            self.accordions.accesscontrol.is_opened and
+            (self.accordions.accesscontrol.tree.selected_item.text ==
```

Comment:
No need for extra parentheses, indentation is enough to visually connect the expression on two lines.
The ``return`` statement already has parentheses so additional ones are not needed for line continuation.

Reply:
Updated to remove the parenthesis. It was originally copy/pasted from another line of code that was longer than 100 chars which used them to wrap it.